content/pubs/pubs.bib

```diff
@@ -1,3 +1,19 @@
+@article{Wang2019,
+title = {Joint Embedding of Graphs},
+author = {Wang, Shangsi and Arroyo, Jesus and Vogelstein, Joshua T. and Priebe, Carey E.},
+journal = {Transactions on Pattern Analysis and Machine Intelligence},
+url = {http://arxiv.org/abs/1703.03862},
```

Comment:
What's the process to update the link when it's "fully" published?

Reply:
currently nonexistent
@jovo recommends writing "in press" or "accepted" in place of a volume/issue/page number

right now we're just periodically checking to see if the articles are in the journal yet, but that seems like unnecessary overhead

Reply:
okay, let's do that please?

even if it's not visible on the website, it will be easy to see which ones need updating later if it's in a field in the bib file.

Reply:
done